app/form_objects/groups/confirm_upgrade_form.rb

```diff
@@ -0,0 +1,29 @@
+module Groups
+  class ConfirmUpgradeForm < ConfirmActionForm
+    include Rails.application.routes.url_helpers
+
+    attr_accessor :group, :current_user, :host
+
+    def submit
+      group.active!
+      send_notification_emails
+    end
```

Comment:
Did you want to pass in the submit-specific values in this PR, or in a later one that covers all our form objects?